llvm/test/tools/llvm-cxxfilt/quote.test

```diff
@@ -1,21 +1,24 @@
-// Show that llvm-cxxfilt --quote can emit quoted demangled symbols (even at the
-// end of a line) but not without double-quoting in the case they are already
-// quoted.
-RUN: echo _Z3barv '"_Z3barv"' > %t
-RUN: echo '"_Z3barv"' _Z3barv >> %t
-// This is not mangled, thus it should not be quoted
-RUN: echo 'log()' >> %t
-// Check that an "import thunk for" prefix can be quoted along the demangled name
-RUN: echo __imp__ZSt6futureIvE >> %t
-RUN: llvm-cxxfilt --quote < %t | FileCheck --match-full-lines --check-prefix=CHECK-FILE %s
+// Show that llvm-cxxfilt --quote adds quotes around demangled symbols, unless the symbol is already quoted.
```

Comment:
Nit: sorry, thought I said this, but please reflow comments to approximately 80 characters, just like in regular code.

Reply:
This is my usual coding style actually, so I love this.